### PR TITLE
Custom favicons color

### PR DESCRIPTION
**Pull request by klaidliadon**
Added the ability of customizing favions colors

### CODE REVIEW

File: assets/scss/_pkg.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.pkg {
```
Comment (rhazdon) [inline assets/scss/_pkg.scss]: Where is this used?

File: assets/scss/_fonts.scss [hunk 2]
```diff
@@ -1,50 +1,44 @@
 @font-face {
-  font-family: 'Inter UI';
```
Comment (rhazdon) [inline assets/scss/_fonts.scss]: Why did you change the font?

File: assets/scss/_pkg.scss (hunk 1 above)
Comment (klaidliadon) [reply assets/scss/_pkg.scss]: You can ignore that part, I accidentally pushed changes that should not be in the PR.

File: assets/scss/_fonts.scss (hunk 2 above)
Comment (klaidliadon) [reply assets/scss/_fonts.scss]: See https://github.com/rhazdon/hugo-theme-hello-friend-ng/pull/62#discussion_r327017467

**Comment by klaidliadon** [comment]
@rhazdon The incriminated commits were removed

**Comment by rhazdon** [comment]
Thank you, it looks good now. :) 

Could you please solve the merge conflict? I will then push the button. :)
